extra_tests/snippets/stdlib_threading.py

```
@@ -0,0 +1,25 @@
+import threading
+
+
+def run_exec(code):
+    result = {}
+
+    def worker():
+        scope = {"__builtins__": __builtins__}
+        exec(code, scope, scope)
```

Ruff (0.15.9)

[error] Use of `exec` detected

(S102)